### PR TITLE
fix: Allow contribution to videos => Allow contributions for videos

### PR DESCRIPTION
**Pull request by AnudeepGunukula**
Resolves the issue #16986
This pr will solve the issue https://github.com/brave/brave-browser/issues/16986

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: components/resources/strings/brave_components_resources_en-GB.xtb [hunk 1]
```diff
@@ -200,7 +200,7 @@
 <translation id="7286722244752781970">Minimum page time before logging a visit</translation>
 <translation id="4424486208397124985">Minimum visits for publisher relevancy</translation>
 <translation id="6248554379744796526">Show non-verified sites in list</translation>
-<translation id="5875669387851353511">Allow contribution to videos</translation>
+<translation id="5875669387851353511">Allow contributions for videos</translation>
```
Comment (deepxcode) [inline components/resources/strings/brave_components_resources_en-GB.xtb:203]: No need to make changes to this file, you can revert this changes

**Comment by deepxcode** [comment]
@AnudeepGunukula  can you revert the change to` app/resources/brave_generated_resources_en-GB.xtb`? We only need the change to `app/brave_generated_resources.grd`

Basically, the Brave team will upload strings to a vendor and then download translations. Any changes to the XTB files will get overwritten.
This was told to me by @bsclifton.

**Comment by AnudeepGunukula** [comment]
Hi @deepxcode . Thankyou for your inputs 
I had reverted back changes.

**Comment by bsclifton** [comment]
Something weird happened here when I tried to rebase; I'll try to find your commit and resubmit

**Comment by bsclifton** [comment]
Looks like this failed because it's already been changed 😄
